compendium/modules/w04-objects-exercise.tex

```diff
@@ -485,17 +471,15 @@
 \end{minipage}
 
 \begin{REPL}
-> scala -cp introprog_2.13-1.1.4.jar
+> scala -cp introprog_2.13-1.1.4.jar -repl
```

Comment:
should be updated eventually; issue here: https://github.com/lunduniversity/introprog/issues/561